modules/end-user-portal/src/Pages/Search/Results.cshtml

```diff
@@ -1,100 +1,56 @@
 @page
-@using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers
-@using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers.Constants
+@using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers.Enumerations
 @model GpConnect.NationalDataSharingPortal.EndUserPortal.Pages.Search.ResultsModel
 
+@inject Microsoft.AspNetCore.Http.IHttpContextAccessor HttpContextAccessor
```

Comment:
Is line 5 used?

Reply:
I've shifted this code into the Model so the UI code is a bit cleaner

Reply:
Thank you